spec/system/assigning/viewing_your_assigned_applications_spec.rb

```diff
@@ -89,13 +89,13 @@
     end
   end
 
-  context 'when using the all applications link' do
+  context 'when using the check all applications link' do
     before do
-      click_on('View all open applications')
+      click_on('Check all open applications')
     end
 
     it 'proceeds to the correct page' do
-      expect(page).to have_content 'All open applications'
+      expect(page).to have_content 'Check all open applications'
```

Comment:
So this is not the link that you've changed. This is the title of the page that the link goes to. 

If you read the test, it goes:
Test to check use of check all applications link
Click on 'check all applications'
It goes to the right page
Check it goes to the right page by checking the title of the page we got redirected to